pages/data.js

```diff
@@ -228,7 +228,7 @@ export default class Explore extends React.Component {
                     />
                   )}
                 </div>
-              ))};
+              ))}
```

Comment:
Snuck this change in to remove a semicolon that was inadvertently being rendered on the page.

Reply:
Ha!